tests/integration/concrete/factory/create-merkle-instant/createMerkleInstant.t.sol

```diff
@@ -52,7 +52,7 @@ contract CreateMerkleInstant_Integration_Test is Integration_Test {
         );
 
         // It should create the campaign with custom fee.
-        assertEq(actualInstant.FEE(), customFee, "fee");
+        assertEq(actualInstant.MINIMUM_FEE(), customFee, "fee");
```

Comment:
let's say `"custom fee"` in the comment.

Reply:
https://github.com/sablier-labs/airdrops/pull/68/commits/5f6b5e66da41cc7ce2fc2333a8deeb3d16fb6174